VideoLocker/src/main/java/org/edx/mobile/loader/CoursesAsyncLoader.java

```diff
@@ -34,7 +35,7 @@ public CoursesAsyncLoader(Context context, Bundle args){
     public AsyncTaskResult<List<EnrolledCoursesResponse>> loadInBackground() {
 
         Api api = new Api(getContext());
-        if (this.oauthToken == null){
+        if (Config.getInstance().getSocialSharingConfig().isEnabled() && this.oauthToken == null) {
```

Comment:
Fixing this is really necessary once we turn on the social feature. However, at this moment, I think we should just merge this fix. This seems a bigger change and may cause lot of regression.